### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 1 - Esercizio 21

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#478] [Esercizi M-Z]
![foglio 1 es.21](https://github.com/user-attachments/assets/0fc05c0e-6f60-48f3-80cc-b93a9e134cb8)

**Comment by marcoseverini** [comment]
<img width="873" alt="immagine" src="https://github.com/user-attachments/assets/347e9505-e7fc-43be-a41e-824d0573047d">

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/29aa055b-5d02-4285-a651-f18f094b1d16)
vera dimostrazione
